### PR TITLE
anguy particles

### PR DESCRIPTION
**Pull request by anguy134**
halp. segfaults after rebasing. particles are wack. 
https://github.com/bbeck13/RhythmRunner/issues/91

### CODE REVIEW

File: src/game_renderer/GameRenderer.h [hunk 1]
```diff
@@ -37,6 +38,7 @@ class GameRenderer {
   static std::unordered_set<std::shared_ptr<GameObject>>* GetObjectsInView(
       std::shared_ptr<std::vector<glm::vec4>> vfplane,
       std::shared_ptr<Octree> tree);
+  ParticleGenerator *Particles;
```
Comment (josepharhar) [inline src/game_renderer/GameRenderer.h]: This shouldn't be a pointer, remove the *

File: src/game_renderer/GameRenderer.cpp [hunk 2]
```diff
@@ -206,6 +207,8 @@ void GameRenderer::RenderMinimap(GLFWwindow* window,
   std::shared_ptr<GameCamera> camera = game_state->GetCamera();
   std::shared_ptr<Player> player = game_state->GetPlayer();
   std::shared_ptr<Sky> sky = game_state->GetSky();
+  Particles = new ParticleGenerator(5000);
+  RendererSetup::PreRender(window);
```
Comment (josepharhar) [inline src/game_renderer/GameRenderer.cpp]: delet this

**Comment by alexottoboni** [comment]
I'm going to make some changes to this before we merge it.

File: assets/shaders/shadowmap_frag.glsl [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#version 330 core
```
Comment (alexottoboni) [inline assets/shaders/shadowmap_frag.glsl:1]: Ah, I just noticed that shadow mapping is kind of started in here.

**Comment by bbeck13** [comment]
@alexottoboni what changes where you going to make to this I'm 🆗 with this going on master as it currently is
CR 🆗 QA 🙆

**Comment by alexottoboni** [comment]
CR 🥇

**Comment by alexottoboni** [comment]
QA 👍

**Comment by josepharhar** [comment]
CR :100: QA :100:

**Pull request merged**